data_collection/gazette/spiders/rj/rj_resende.py

```diff
@@ -0,0 +1,56 @@
+import re
+from datetime import date
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjResendeSpider(BaseGazetteSpider):
+    name = "rj_resende"
+    TERRITORY_ID = "3302254"
```

Comment:
Parece que esse tá errado de acordo com https://github.com/okfn-brasil/querido-diario/blob/main/data_collection/gazette/resources/territories.csv

```suggestion
    TERRITORY_ID = "3304201"
```